### DISCUSSION TITLE
Computer Enthusiast

### DISCUSSION THREAD

**Discussion by atuneforu** [community/community#146708] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

WiFi and Flipper Zero related projects interest me.

**Comment by atuneforu** [comment]
Hey guys trying to learn my way around GitHub if anyone out there knows of any new Flipper Zero projects please do not hesitate to give me a heads up it would rewarded. 🙏

**Comment by CyberSphinxxx** [comment]
Hey @atuneforu, If you're exploring Flipper Zero projects, you should definitely check out this repository: [A collection of awesome resources for the Flipper Zero device](https://github.com/djsime1/awesome-flipperzero). It includes a comprehensive list of tools, guides, and projects related to the Flipper Zero.

![Opera Snapshot_2024-12-10_115024_github com](https://github.com/user-attachments/assets/fe3e8198-7a55-4ec8-94b3-68e9d9ee0102)
